pyproject.toml

```diff
@@ -7,6 +7,7 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.9"
+ruff = "^0.0.254"
```

Comment:
Think this needs to be a dev-dependency.

Reply:
@MrPowers fixed.